docs/semgrep-code/semgrep-pro-engine-intro.md

```diff
@@ -157,18 +157,11 @@ rules:
 
 <dl>
     <dt>Cross-file</dt>
-    <dd>The cross-file analysis (also called interfile analysis) gathers context across multiple files to help security engineers deeply understand their organization's security issues. Semgrep Pro Engine reduces noise and detects new vulnerabilities that Semgrep OSS Engine can't find. Cross-file analysis runs on nightly scans. These scans may take longer to complete and can use more memory than Semgrep OSS Engine scans. See the available languages for cross-file (interfile) analysis in <a href="/docs/supported-languages/#semgrep-pro-engine">Semgrep Pro Engine supported languages</a>.</dd>
+    <dd>The cross-file analysis (interfile analysis) looks across multiple files to help security engineers find more of their organization's security issues. Semgrep Pro Engine reduces noise and detects new OWASP Top 10 vulnerabilities that Semgrep OSS Engine can't find. By default, cross-file analysis runs on PRs and nightly scans. These scans may take longer to complete and can use more memory than Semgrep OSS Engine scans. See the available languages for cross-file (interfile) analysis in <a href="/docs/supported-languages/#semgrep-pro-engine">Semgrep Pro Engine supported languages</a>.</dd>
```

Comment:
- We can skip the opening "The" for these definitions.
- "Cross-file analysis looks across multiple files..."
    - May I ask why we changed "gathers context" into "looks across"? Because when we scan with Semgrep, with or without cross-file analysis, we "look" at all the files anyway, the differentiating factor is to take those files **in concert**, or more precisely, that we track a function or variable or object, that may have been defined in one file, to its use in another file.
    - "Look across" in this case isn't as clear as "gather context" or "examine multiple files in concert".